__tests__/app.test.js

```diff
@@ -164,4 +164,104 @@ describe('app()', () => {
       });
     });
   });
+  describe('PATCH', () => {
+    describe('/api/articles/:article_id', () => {
+      it('200: should respond with a status of 200 and update the value of votes accordingly', () => {
+        const newVote = { inc_votes: 500 };
+        return request(app)
+          .patch('/api/articles/1')
+          .send(newVote)
+          .expect(200)
+          .then(({ body }) => {
+            const { article } = body;
+            expect(article.article_id).toBe(1);
+            expect(article).toHaveProperty('title', expect.any(String));
+            expect(article).toHaveProperty('topic', expect.any(String));
+            expect(article).toHaveProperty('author', expect.any(String));
+            expect(article).toHaveProperty('body', expect.any(String));
+            expect(article).toHaveProperty('created_at', expect.any(String));
+            expect(article).toHaveProperty('votes', 500);
+            expect(article).toHaveProperty(
+              'article_img_url',
+              expect.any(String)
+            );
+          });
+      });
+      it('200: should respond with a status of 200 and update the value of votes accordingly', () => {
+        const newVote = { inc_votes: -100 };
+        return request(app)
+          .patch('/api/articles/1')
+          .send(newVote)
+          .expect(200)
+          .then(({ body }) => {
+            const { article } = body;
+            expect(article.article_id).toBe(1);
+            expect(article).toHaveProperty('title', expect.any(String));
+            expect(article).toHaveProperty('topic', expect.any(String));
+            expect(article).toHaveProperty('author', expect.any(String));
+            expect(article).toHaveProperty('body', expect.any(String));
+            expect(article).toHaveProperty('created_at', expect.any(String));
+            expect(article).toHaveProperty('votes', -100);
+            expect(article).toHaveProperty(
+              'article_img_url',
```

Comment:
As we know what the article_id is here, we can expect the exact data rather than expect.any(String)